raiden/ui/cli.py

```diff
@@ -90,12 +97,61 @@ def toggle_trace_profiler(raiden):
         raiden.profiler.start()
 
 
+def check_json_rpc(client):
+    try:
+        client_version = client.call('web3_clientVersion')
+    except (requests.exceptions.ConnectionError, JSONRPCClientReplyError):
+        print(
+            "\n"
```

Comment:
SCan you change this and all other string literlas to be with single quotes?

Reply:
Humm, the next line actually uses a single quote, I would like to not quote that and keep these quotes consistent (all double quotes), do you really want to change it?

Reply:
no it's fine then.